### PR TITLE
Changed humburger icon by using drawer toggle

### PR DESCRIPTION
**Pull request by konifar**
# Summary

Changed humburger icon by using `ActionBarDrawerToggle`.
I refered the tutorial below.
https://developer.android.com/training/implementing-navigation/nav-drawer.html

With `ActionBarDrawerToggle`, humburger icon is animated when drawer is opened or closed.
# Screen shot

2015-06-29_18_01_53_png_ _2015-06-29_18_24_30_png

### CODE REVIEW

File: SampleDesignSupportLibrary/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -17,4 +19,8 @@
         <item name="windowActionModeOverlay">true</item>
     </style>
 
+    <style name="AppDrawerArrowToggle" parent="Base.Widget.AppCompat.DrawerArrowToggle">
+        <item name="color">@color/txt_reverse</item>
```
Comment (konifar) [inline SampleDesignSupportLibrary/app/src/main/res/values/styles.xml:23]: I defined `DrawerArrowToggle` style to change humburger icon color.
http://stackoverflow.com/questions/26439572/how-to-style-the-drawerarrowtoggle-from-android-appcompat-v7-21-library

**Comment by kgmyshin** [comment]
:man_with_gua_pi_mao: :+1: 

@ Tazawa-san, I'd like you to decide this.

**Comment by tazake** [comment]
LGTM :+1:

**Comment by konifar** [comment]
@Tazake @kgmyshin Thanks for your quick review! :)

**Pull request merged**